src/ezmsg/util/perf_test.py

```diff
@@ -91,8 +93,8 @@ class LoadTestSubscriberState(ez.State):
 
 class LoadTestSubscriber(ez.Unit):
     INPUT = ez.InputStream(LoadTestSample)
-    SETTINGS: LoadTestSettings
-    STATE: LoadTestSubscriberState
+    SETTINGS =LoadTestSettings
```

Comment:
Should `SETTINGS` and `STATE` really be "=" here? I thought they were supposed to be `:`?

Reply:
Ah, I see this is the topic of #135

Reply:
Apologies, this was not supposed to creep into this PR.